components/base/sensorcontrolled/spin.go

```diff
@@ -21,7 +21,7 @@ const (
 // Spin commands a base to turn about its center at a angular speed and for a specific angle.
 func (sb *sensorBase) Spin(ctx context.Context, angleDeg, degsPerSec float64, extra map[string]interface{}) error {
 	sb.stopLoop()
-	if int(angleDeg) >= 360 {
+	if math.Abs(angleDeg) > 330.0 {
```

Comment:
ah, any way we could keep this a 360 bound? if not just add a TODO for me and a ticket.

Reply:
oh oops yep meant to change that back